### PR TITLE
Import feb 2020 data

### PR DESCRIPTION
**Pull request by MahmudH**
The latest version of data, Feb 2020, has been uploaded to S3. This script is updated in order to download the new data.

Trello: https://trello.com/c/6ss0MKQw/1783-8-import-new-mapit-data-using-docker.

### CODE REVIEW

File: import-db-from-s3.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 set -e
 
 # get the data from s3
-sudo -u deploy curl 'https://s3-eu-west-1.amazonaws.com/govuk-custom-formats-mapit-storage-production/source-data/2019-08/mapit-may-2019-august-update.sql.gz' -o mapit.sql.gz
-if ! echo "f13a7431ed55fb818c89f959d5570d92289ba525 mapit.sql.gz" | sha1sum -c -; then
+sudo -u deploy curl 'https://s3-eu-west-1.amazonaws.com/govuk-custom-formats-mapit-storage-production/source-data/2020-02/mapit-march-2020-feb-update.sql.gz' -o mapit.sql.gz
+if ! echo "8a792e4d7dbed15e6d1a392aa9e82ec64de91333 mapit.sql.gz" | sha1sum -c -; then
```
Comment (AlanGabbianelli) [inline import-db-from-s3.sh]: How can I review/verify this code?

Comment (MahmudH) [reply import-db-from-s3.sh]: You can check the location of the file in S3, then download it and do a checksum. That will verify the URL and checksum

Comment (issyl0) [reply import-db-from-s3.sh]: Aside: Why are we using sha1?

Comment (MahmudH) [reply import-db-from-s3.sh]: I'm not sure the reason why but this is part of the Mapit documentation

**Pull request merged**